### PR TITLE
Add development docs to setup Kibana and Elastic Agent

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,21 +6,63 @@
...
+
+```
+# Start ES
+yarn es snapshot -E xpack.security.authc.api_key.enabled=true
+
+# Start KB
+yarn start --no-base-path
+```
+
+As soon as all is running, go to `http://localhost:5601`, enter `elastic/changeme` as credential and navigate to Fleet. Trigger the Fleet. As soon as this is completed, copy the `policy id` and `enrollment token` for the fleet-server policy. The policy id can be copied from the URL, the enrollment token can be found in the Enrollment Token list.
```
Comment (nchaulet) [inline README.md:40]:
```suggestion
As soon as all is running, go to `http://localhost:5601`, enter `elastic/changeme` as credential and navigate to Fleet. Trigger the Fleet setup. As soon as this is completed, copy the `policy id` and `enrollment token` for the fleet-server policy. The policy id can be copied from the URL, the enrollment token can be found in the Enrollment Token list. 
```

File: README.md [hunk 2]
```diff
@@ -6,21 +6,63 @@
 
 fleet-server is under development. The following are notes to help developers onboarding to the project to quickly get running. These notes might change at any time.
 
-### Startup fleet-server
+## Setup
 
-Currently to startup fleet-server, the Kibana encryption key is needed. There are two options for this.
+To run and test fleet-server, a recent version of Elastic Agent and Kibana are needed. In the following Elastic Agent and Kibana are built from source. The fleet-server itself is not built from source but pulled from the latest snapshot build. It would be possible to also pull Elastic Agent or Kibana from the latest snapshot but the assumption that is made here that whoever is testing this, is likely developing either Elastic Agent or on the Kibana side.
```
Comment (blakerouse) [inline README.md:11]: If the expectation is that you are developing with Elastic Agent or Kibana, why not also the Fleet Server? I think we should make it consistent and have this include how to build Fleet Server.

It really is as simple as `make release` inside of the Fleet Server repository.

Comment (ruflin) [reply README.md:11]: `make release` and then copy it over to the right place I assume?

Comment (blakerouse) [reply README.md:11]: Yes, it needs to be copied.

Comment (ruflin) [reply README.md:11]: @blakerouse Would you be ok to get it in as is and follow up with the fleet-server part?

Comment (blakerouse) [reply README.md:11]: Sure!

**Review by blakerouse — APPROVED** [review]

**Pull request merged**